src/nvim/move.c

```diff
@@ -1345,20 +1309,15 @@ void scroll_cursor_top(int min_scroll, int always)
   new_topline = top + 1;
 
   /* count filler lines of the cursor window as context */
-  i = diff_check_fill(curwin, curwin->w_cursor.lnum);
-  used += i;
-  extra += i;
+  int extra = diff_check_fill(curwin, curwin->w_cursor.lnum);
+  used += extra;
```

Comment:
Dear Nicolas Cornu,

The email sent to danielm@tinyhub.tk with subject
"Re: [neovim] Scope variables in move.c (#3297)"
will be delivered as soon as possible.

Your email address is automatically added to the
recipients 'whitelist'. This means for further
emails to danielm@tinyhub.tk, you will not receive a delivery
notification again.

TK MAILIAS
*************************************************
Looking for a way to Protect Your Identity and 
for a great no-more-spam solution for your email
address reply@reply.github.com? Sign up for TK Mailias.

It's free! Please visit http://www.mail.tk/